src/shafaque/Assignment79_1.java

```diff
@@ -0,0 +1,19 @@
+package shafaque;
+
+public class Assignment79_1 {
+
+	void pattern() {
+		for (int i = 1; i <= 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible